src/_component/portfolio/portfolio.scss

```diff
@@ -0,0 +1,97 @@
+.portfolio {
+  @extend %red-header;
+}
+
+.portfolio__title {
+  @extend %red-box;
+  font-weight: 500;
+}
+
+.portfolio__section {
+  @extend %align-items;
+  position: relative;
+}
+
+.portfolio-first__title,
+.portfolio-second__title {
+  color: #707070;
```

Comment:
All colors should be declared in a sass file - (name that color - Dove Gray)

Reply:
So every time you re-use the color #707070 
color: $color-Dove-Gray

Reply:
Yes, you are right! I forgot to do it but I am going to change it now. Those pictures are only temporary, soon I will change everything with real content. Thank you for your review!